_cloudstor/01-overview.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Overview
+order: 1
+duration: 5
+---
+
+### What is AARNET CloudStor
```

Comment:
I think the proper way of writing this is 'AARNet'. Please change all instances of AARNet to use this convention. Also you need to fix any uses of 'CloudStor' that aren't like this.